### ISSUE TITLE
[ dotnet-8.0.200-rtm.24069.25] dotnet tool install command doesn't show install successful message

### ISSUE THREAD

**Issue by Junjun-zhao** [dotnet/sdk#38355]
**Application Name:** globaltools
**OS**： Windows 10 21H2
**CPU**: X64
**.NET Build Number**: dotnet-8.0.200-rtm.24069.25

**Verify Scenarios**

1. Windows10 21h2 x64 + dotnet-sdk-8.0.101: Pass
2. Windows10 21h2 x64 + dotnet-sdk-8.0.102: Pass
3. Windows10 21h2 x64 + dotnet-sdk-8.0.200: Fail

**Issue Description:**

When installing dotnet tools with "dotnet tool install" command, it doesn't show the installation successful message on dotnet-sdk-8.0.200.

**Repro steps:**

The machine has dotnet-sdk-8.0.200 installed.

1.Open command prompt window.
2. Input "```dotnet tool install --global MarkdownSnippets.Tool```" command.
3. Press "Enter"

**Expected Result:**
Tool will be installed successfully with following message shows:
![image](https://github.com/dotnet/sdk/assets/83067197/ee28f025-ed19-49f5-9fd8-47e5a7eef419)

**Actual Result:**
Tool is installed successfully but no message shows.
![image](https://github.com/dotnet/sdk/assets/83067197/cbfe59e2-73f2-4725-9865-59ba4e8786ea)

```
C:\Users\Appcompat>dotnet --info

.NET SDK:
 Version:           8.0.200
 Commit:            438cab6a9d
 Workload version:  8.0.200-manifests.cdf2cc8e

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19044
 OS Platform: Windows
 RID:         win-x64
 Base Path:   C:\Program Files\dotnet\sdk\8.0.200\

.NET workloads installed:
There are no installed workloads to display.

Host:
  Version:      8.0.2
  Architecture: x64
  Commit:       1381d5ebd2

.NET SDKs installed:
  8.0.200 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 8.0.2 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 8.0.2 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 8.0.2 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
```

**Findings:**

File this bug to track on .NET 8 build, the same error occurred on the .NET 9.0: [[9.0.100-alpha.1.24055.6] dotnet tool install command doesn't show install successful message · Issue #37879 · dotnet/sdk (github.com)](https://github.com/dotnet/sdk/issues/37879) 

@dotnet-actwx-bot @dotnet/compat

**Comment by JL03-Yue** [comment]
Thank you for reporting. We are currently fixing it in #38277

**Comment by JL03-Yue** [comment]
Thank you for reporting. The issue is fixed in #38277.

**Comment by Junjun-zhao** [comment]
Verified with dotnet-sdk-8.0.202-servicing.24116.49, this issue has been fixed. Closing this issue
